addons/crm_livechat/models/crm_lead.py

```diff
@@ -1,23 +1,31 @@
 from odoo import api, fields, models
+from odoo.exceptions import AccessError
 
 
 class CrmLead(models.Model):
     _inherit = "crm.lead"
 
-    channel_id = fields.Many2one(
+    livechat_channel_id = fields.Many2one(
         "discuss.channel",
         "Live chat from which the lead was created",
         readonly=True,
-        groups="base.group_erp_manager",
+        groups="sales_team.group_sale_salesman",
```

Comment:
Is there any point to keep a group if the access is restricted through overrides?

Reply:
Notably portal users may gain read access on leads, they are going to have crashes without that group.

Reply:
It's restricted not to allow random access to any channels, still those fields only make sense for sales ppl.

Reply:
Public and portal users can read discuss channel with standard ACL, so I don't think it would crash, unless I'm missing something?

Reply:
Maybe if it was not a discussion with themselves? For example discussion with visitor, lead assigned to portal user?

But if we only check existence of channel it should be fine no? m2o permits it. Only if reading the channel table it would crash, but I don't think we read it in this flow.